exercises/practice/book-store/book-store_spec.lua

```diff
@@ -76,12 +76,11 @@ describe('book-store', function()
     assert.are.same(10240, book_store.total(basket))
   end)
 
-  -- LuaFormatter off
-  it('check that groups of four are created properly even when there are more groups of three than groups of five', function()
+  it('check that groups of four are created properly even when there are more groups of three than groups of five',
+     function()
     local basket = { 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 4, 4, 5, 5 }
     assert.are.same(14560, book_store.total(basket))
   end)
-  -- LuaFormatter on
```

Comment:
I re-generated all specs to make sure that we hadn't inadvertently included tests that shouldn't have been included. When I did this it reverted the change that was made in https://github.com/exercism/lua/pull/513 to fix an issue with the test runner that has since been resolved.